docs/sources/datasources/alertmanager.md

```diff
@@ -29,7 +29,7 @@ datasources:
     url: http://localhost:9093
     access: proxy
     jsonData:
-      implementation: 'prometheus' # alternatively 'cortex'
+      implementation: 'prometheus' # alternatively 'mimir'
```

Comment:
Is this correct?

Reply:
I believe this is not correct but @JohnnyQQQQ or @alexweav can correct me if I'm wrong.